### PR TITLE
Allow k6 to run for multiple namespaces

### LINKED ISSUE #8515: Automate K6 In Mainnet Staging for Citus
### Problem

Need to automate k6 tests in staging v2 cluster

### Solution

- K6 tests kick off automatically immediately after v1 run
- Runs web3 and rest k6 tests

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
* update testkube resources to support k6 tests in multiple namespaces

**Related issue(s)**:

Fixes #8515 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-mirror-common/templates/test-rest.yaml [hunk 1]
```diff
@@ -23,11 +25,12 @@ spec:
       - mapToVariables: true
         mount: false
         reference:
-          name: {{ include "hedera-mirror-common.fullname" . }}-test-rest
+          name: {{ include "hedera-mirror-common.fullname" $ }}-test-rest  # Todo verify sharing the config is ok
```
Comment (jnels124) [inline charts/hedera-mirror-common/templates/test-rest.yaml:28]: Do we want to share k6 config between namespaces?

File: charts/hedera-mirror-common/values.yaml [hunk 2]
```diff
@@ -388,9 +388,9 @@ testkube:
     extraExecutionRequestVariables: {}
     gitBranch: ""  # Default to .Chart.AppVersion
     schedule: ""  # Cron schedule of the test suite, default to no schedule
-    target:
-      namespace: ""
-      release: mirror
+    targets:
+      - namespace: ""
+        release: mirror
```
Comment (steven-sheehy) [inline charts/hedera-mirror-common/values.yaml:393]: Probably we should have this as an empty list?
```suggestion
    targets: []
    #  - namespace: ""
    #    release: mirror
```

File: charts/hedera-mirror-common/templates/test-trigger-rest.yaml [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
+    namespace: {{ $target.namespace }}
     labelSelector:
       matchLabels:
-        test-trigger: rest-trigger
+        test-trigger: rest-trigger-{{ $target.namespace }}
+{{- else }}
+    namespace: {{ $.Values.testkube.namespace }}
+    labelSelector:
+      matchLabels:
+        test-name: {{ $previous := sub $idx 1 | index $.Values.testkube.test.targets }}test-web3-{{ $previous.namespace }}
```
Comment (steven-sheehy) [inline charts/hedera-mirror-common/templates/test-trigger-rest.yaml:22]: Confusing that this is all on one line.
```suggestion
        {{ $previous := sub $idx 1 | index $.Values.testkube.test.targets }}
        test-name: test-web3-{{ $previous.namespace }}
```

File: charts/hedera-mirror-common/templates/test-rest.yaml (hunk 1 above)
Comment (steven-sheehy) [reply charts/hedera-mirror-common/templates/test-rest.yaml:28]: That's probably best for now until we see a need to diverge. Can remove comment.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: charts/hedera-mirror-common/values.yaml (hunk 2 above)
Comment (jnels124) [reply charts/hedera-mirror-common/values.yaml:393]: yes it should be empty

**Review by xin-hedera — DISMISSED** [review]
LGTM

File: charts/hedera-mirror-common/values.yaml [hunk 4]
```diff
@@ -388,9 +388,7 @@ testkube:
     extraExecutionRequestVariables: {}
     gitBranch: ""  # Default to .Chart.AppVersion
     schedule: ""  # Cron schedule of the test suite, default to no schedule
-    target:
-      namespace: ""
-      release: mirror
+    targets: []
```
Comment (steven-sheehy) [inline charts/hedera-mirror-common/values.yaml:391]: I think keeping the structure of the target object as a comment would make it easier to configure. This is what I recommended in my previous comment.

**Review by xin-hedera — APPROVED** [review]

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**